metrics-observer/src/metrics.rs

```diff
@@ -16,11 +16,8 @@ mod proto {
     include!(concat!(env!("OUT_DIR"), "/event.proto.rs"));
 }
 
-use self::proto::{
-    event::Event,
-    metadata::{Description as DescriptionMetadata, MetricType, Unit as UnitMetadata},
-    Event as EventWrapper,
-};
+use self::proto::metadata::MetricType;
+use proto::event::Event as ProtoEvent;
```

Comment:
Not clear why we're importing `Event` as an aliased name here? There's no conflict I can see.